hypha/apply/funds/templates/funds/application_base.html

```diff
@@ -47,6 +47,7 @@ <h3>{% blocktrans %}Sorry this {{ page|verbose_name }} is not accepting applicat
                     {% endif %}
                 {% endif %}
             {% endfor %}
+            <input  class="link link--button-tertiary" type="submit" value="Save Draft" name="draft" />
```

Comment:
I prefer to use "button" tag throughout unless there are good reason to use "input". I know "input" is in use in many places but my plan is to one day clean it up throughout.

Also all draft buttons should have "formnovalidate" added. mora about that in separate comment.

Reply:
Done.